### PR TITLE
Finished story 1 and started story 2

### PR DESCRIPTION
**Pull request by Osama-Tahboub1**
Finished story 1 and started story 2.

### CODE REVIEW

File: cms/tests/functions.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
...0=>["id"=>"1","projectTitle"=>"Pilot Shop","projectTitleLink"=>"https://dev.maydenacademy.co.uk/students/2018/osama/PilotShopSite/","projectImage"=>"https://dev.maydenacademy.co.uk/students/2018/osama/Portfolio/assets/PilotShop.png"],1=>["id"=>"2","projectTitle"=>"CMS Project","projectTitleLink"=>"https://dev.maydenacademy.co.uk/students/2018/osama/PilotShopSite/", "projectImage"=>"https://dev.maydenacademy.co.uk/students/2018/osama/Portfolio/assets/PilotShop.png"],2=>["id"=>"3","projectTitle"=>"New Project","projectTitleLink"=> "https://dev.maydenacademy.co.uk/students/2018/osama/PilotShopSite/","projectImage"=>"https://dev.maydenacademy.co.uk/students/2018/osama/Portfolio/assets/PilotShop.png"],3=>["id"=>"4","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],4=>["id"=>"5","projectTitle"=> NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],5=>["id"=>"6","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],6=>["id"=>"7","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],7=>["id"=>"8","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],8=>["id"=>"9","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL]];
```
Comment (nsantiagoblair) [inline cms/tests/functions.php]: Don't need the `0` in an indexed array

File: cms/portfolio.php [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        <form method="post" action="portfolio.php">
+            <textarea name="projectId9" cols="20" rows="5" maxlength="500"><?php echo returnProjectId($projectInfo, 9); ?></textarea>
+            <textarea name="projectTitle9" cols="20" rows="5" maxlength="500"><?php echo returnProjectTitle($projectInfo, 9); ?></textarea>
+            <textarea name="projectTitleLink9" cols="100" rows="5" maxlength="500"><?php echo returnProjectTitleLink($projectInfo, 9); ?></textarea>
+            <textarea name="projectImage9" cols="100" rows="5" maxlength="500"><?php echo returnProjectImage($projectInfo, 9); ?></textarea>
+            <p></p>
+            <input type="submit" value="Update">
+        </form>
+    </div>
+</body>
```
Comment (nsantiagoblair) [inline cms/portfolio.php]: This forms could be put into functions that return a string of the form to clean up your index page

File: cms/portfolio.php [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+if ($projectTitle1 != NULL && $projectTitleLink1 != NULL && $projectImage1 != NULL && $projectId1!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle1, $projectTitleLink1, $projectImage1, $projectId1);
+    echo $updatedInfo;
+
+}
+
+
+$projectTitle2 = $_POST['projectTitle2'];
+$projectTitleLink2 =$_POST['projectTitleLink2'];
```
Comment (nsantiagoblair) [inline cms/portfolio.php]: space after equals

File: cms/portfolio.php [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+$projectTitle3 = $_POST['projectTitle3'];
+$projectTitleLink3 =$_POST['projectTitleLink3'];
+$projectImage3 = $_POST['projectImage3'];
+$projectId3 = $_POST['projectId3'];
+
+if ($projectTitle3 != NULL && $projectTitleLink3 != NULL && $projectImage3 != NULL && $projectId3!= NULL) {
+    $updatedInfo = updateProject($db, $projectTitle3, $projectTitleLink3, $projectImage3, $projectId3);
+    echo $updatedInfo;
+
```
Comment (nsantiagoblair) [inline cms/portfolio.php]: no need for this linebreak

File: cms/functions.php [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+}
+
+/* Doc Block
+ * Returns all project image links from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of project image links in the arrays within the arrays pulled from the database.
+ */
+function returnProjectImages(array $array): string{
```
[the remark this answers is not in the document]
Comment (Osama-Tahboub1) [reply cms/functions.php]: fixed.

File: cms/functions.php [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+}
+
+/* Doc Block
+ * Returns project id's from array content provided by database.
+ *
+ * @param $array array associative array provided by database.
+ *
+ * @return string a list of id's or keys in the arrays within the arrays pulled from the database.
+ */
+function returnProjectIds(array $array): string{
```
[the remark this answers is not in the document]
Comment (Osama-Tahboub1) [reply cms/functions.php]: fixed.

File: cms/portfolio.php (hunk 3 above)
Comment (Osama-Tahboub1) [reply cms/portfolio.php]: fixed.

**Pull request merged**